src/validation/fields/address.validation.ts

```diff
@@ -111,91 +111,6 @@ export const usual_residential_service_address_validations = [
     .custom((value, { req }) => checkInvalidCharactersIfRadioButtonSelected(req.body.is_service_address_same_as_usual_residential_address === '0', ErrorMessages.POSTCODE_ZIPCODE_INVALID_CHARACTERS, value)),
 ];
 
-// Checking only max lengths and invalid characters for BO address fields
```

Comment:
don't we need the address validation anymore?

Reply:
We do need validation, I just removed not used old validation checks replaced by others